### DISCUSSION TITLE
condition example to determine movement direction via zones.

### DISCUSSION THREAD

**Discussion by garywatts** [blakeblackshear/frigate#2523] [Ask A Question]
Hi.
I am trying to get my head around conditions. I currently have 2 driveway zones. I can set up a notification if both entered zones are triggered but is it be possible to trigger the notification if zone one was triggered before zone two but not the other way around. 

I only care to find out if cars are entering the drive, not leaving the drive.

p.s. Thanks for your work on this project.

**Comment by blakeblackshear** [comment]
You would look for a message where zone one was listed in the before data without zone two and both are listed in the after data.
